BadApi/Controllers/AccountController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using BadApi.Controllers.Base;
+using BadApi.Domain;
+using BadApi.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Web;
+using System.Web.Http;
+
+using Dapper;
+using BadApi.Models.Account;
+
+namespace BadApi.Controllers
+{
+	[Authorize]
+	[RoutePrefix("api/account")]
+    public class AccountController : BaseApiController
+    {
+        // GET: Account
+		[AllowAnonymous]
+		[HttpGet]
+		[Route("login")]
+        public async Task<Account> Login([FromUri] LoginModel model)
+        {
+            var connection = GetConnection();
+			var users = await connection.QueryAsync<Account>("SELECT * FROM dbo.Account;");
```

Comment:
1. Select * is an anti-pattern as it can break data binding if the underlying data columns change.
2. Consider using a filter predicate in the query if possible